### ISSUE TITLE
I have some questions about LmkDet/extract_kp_videos.py

### ISSUE THREAD

**Issue by 904763189cy** [theEricMa/OTAvatar_processing#2]
Hello, I’ve encountered a small issue.

code：
filenames = list()
VIDEO_EXTENSIONS_LOWERCASE = {'mp4'}
VIDEO_EXTENSIONS = VIDEO_EXTENSIONS_LOWERCASE.union({f.upper() for f in VIDEO_EXTENSIONS_LOWERCASE})
extensions = VIDEO_EXTENSIONS
for ext in extensions:
    filenames = sorted(glob.glob(f'{opt.input_dir}/**/*.{ext}'))
    filenames = sorted(glob.glob(f'{opt.input_dir}/**/*.{ext}'))
    filenames = sorted(glob.glob(f'{opt.input_dir}/**/*.{ext}'))

My understanding is that I want to get all files with the extensions mp4 and MP4 under the directory specified by opt.input_dir. However, when running this code, sometimes it can find .mp4 files and sometimes it cannot. I suspect there might be a bug in the code. So I modified the code to only search for mp4 files. One thing I don’t understand is why this line of code is executed three times: filenames = sorted(glob.glob(f’{opt.input_dir}/**/*.{ext}'))

Appreciate if you could clarify my doubts

**Comment by theEricMa** [comment]
We have encountered an intermittent issue where the code is sometimes unable to locate .mp4 files even though they exist. We initially thought it might be a problem with our computer system - after running the code multiple times, it would go from not finding the files to locating all of them. We modified the code to search for the files several times, but this does not appear to have resolved the problem. It seems the root cause is not our system randomly being unable to locate files, but rather something else preventing the code from consistently finding existing .mp4 files. More investigation is needed to understand what is causing the code to sporadically fail to locate these files.
